PalasoUIWindowsForms.Tests/ClearShare/CreativeCommonsLicenseTests.cs

```diff
@@ -112,6 +112,22 @@ public void HasChanges_CanToggle()
 			l.HasChanges = true;
 			Assert.IsTrue(l.HasChanges);
 		}
+		[Test]
+		public void GetDescription_NoTranslation_GivesEnglish()
+		{
+			var l = new CreativeCommonsLicense(true, true, CreativeCommonsLicense.DerivativeRules.DerivativesWithShareAndShareAlike);
+			Assert.AreEqual(l.GetDescription("en"), l.GetDescription("qx"));
 
+		}
+		[Test]
+		public void GetDescription_DontHaveFirstChoice_GivesEnglish()
+		{
+			var l = new CreativeCommonsLicense(true, true, CreativeCommonsLicense.DerivativeRules.DerivativesWithShareAndShareAlike);
+			string usedLanguage;
+			//Assert.AreEqual(l.GetDescription("en"), l.GetDescription(new string[] { "qx", "en" }, out usedLanguage));
+			l.GetDescription(new string[] {"qx", "en"}, out usedLanguage);
+			Assert.AreEqual("en", usedLanguage);
+
+		}
```

Comment:
I agree about the other test idea. Unfortunately I think it would take about a day to modify l10nsharp to be unit-test friendly. Sigh.